dpe/src/commands/certify_key.rs

```diff
@@ -55,11 +55,12 @@ impl<C: Crypto> CommandExecution<C> for CertifyKeyCmd {
                         &self.label,
                         b"ECC",
                     )
+                    .unwrap()
```

Comment:
We should avoid calling unwrap wherever possible (here and in sign.rs). In general, it is prefferable to bubble up the result. Since we're changing error types, you can do `.map_err(|_| <some_dpe_error>)`. This will make `priv_key` a result, but you can always add`?` to the end of the assignment to say "If the result is an error, return it"